_opensource/LUJS.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: LUJS
-status: Active Development
+status: Archive
 language: JavaScript
 desc: Event-driven NodeJS server
 links: [['GitHub', 'https://github.com/RaineBannister/LUJS/', 'fab fa-github']]
-devs: Raine
+author: Raine
 ---
```

Comment:
This should probably just be removed since the repo was made private

Reply:
Agreed, done